.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Type of change
+
+_Select the type of your PR_
+
+- [ ] Bugfix
+- [ ] Enhancement / new feature
+
+
+### Description
+
+_Please describe your pull request_
+
+### Checklist
+
+_Please go through this checklist and make sure all aplicable tasks have been done_
+
+- [ ] Write tests!
```

Comment:
What's so exciting about writing tests that it's deserving of an exclamation mark? ;-) Maybe something a little less broad would make focus people's minds, too: "Updated and/or added new tests to cover new and changed functionality"

Reply:
All the points have exclamation marks ... it is just a typography thing. I will remove them.

Reply:
I do not think that we need to write long stories about what which point means. It is just a checklist ... so IMHO _Write tests_ is sufficient and everyone would understand it.